docs/en/index.rst

```diff
@@ -9,7 +9,7 @@ Project's ROOT directory (where the **composer.json** file is located)
 
 .. code-block:: shell
 
-    php composer.phar require cakephp/authorization:^2.0
+    php composer.phar require "cakephp/authorization:^2.0"
```

Comment:
It should be enough to quote the version, no?

Reply:
It would be, AFAIK it's just a matter of preference. I grew accustomed to this variant as the concatenated constraint is what's promoted in the composer docs.

Reply:
Sure, but I dont know.
This only is relevant if that char is involved, e.g. dev-master and co references are all fine as well.
Adding quoting docs for all cases might be a bit much.

Reply:
I'm not married to it, I'll go with whatever is being preferred, what matters is that people who copy the command get the right version.

Reply:
This form is simple, harder to make mistakes with and will work for all environments.